Resources/Prototypes/_NF/Entities/Clothing/OuterClothing/hardsuits.yml

```
@@ -211,7 +214,8 @@
 - type: entity
   parent: ClothingOuterHardsuitNfsdBronze
   id: ClothingOuterHardsuitNfsdBrigmedic
-  name: NFSD brigmedic patrol hardsuit
+  name: NFSD brigmedic patrol suit
+  description: A lightly armored suit that protects from the danger of space, standard issue to be worn by NFSD on patrol. This specialized variant has brigmedic rank markings.
```

```suggestion
  description: The standard-issue NFSD space proof suit for patrol duty. This variant has brigmedic rank markings.
```
Shorter.  Sheriff wasn't specialized, this isn't specialized.